cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/compute/GcpInstanceResourceBuilder.java

```diff
@@ -361,6 +361,30 @@ public CloudVmInstanceStatus stop(GcpContext context, AuthenticatedContext auth,
         return stopStart(context, auth, instance, true);
     }
 
+    @Override
+    public CloudResource update(GcpContext context, CloudResource resource, CloudInstance cloudInstance,
+        AuthenticatedContext auth, CloudStack cloudStack) throws Exception {
+        String projectId = gcpStackUtil.getProjectId(auth.getCloudCredential());
+        String availabilityZone = cloudInstance.getAvailabilityZone();
+        Compute compute = context.getCompute();
+        String instanceId = cloudInstance.getInstanceId();
+        try {
+            LOGGER.info("Gcp operations are preparing: instanceId: {}, projectId: {}, availabilityZone: {}", instanceId, projectId, availabilityZone);
+            Get get = compute.instances().get(projectId, availabilityZone, instanceId);
+            Instance gcpInstance = get.execute();
+            gcpInstance.setMachineType(String.format("https://www.googleapis.com/compute/v1/projects/%s/zones/%s/machineTypes/%s",
```

Comment:
this url is already present in this class, please consider moving it to a constant